src/app/page.tsx

```diff
@@ -1,113 +1,56 @@
-import Image from "next/image";
+const classCardDiv = 'p-4 mb-5 bg-teal text-white rounded-lg';
+const classCardTitle = 'text-xl font-black';
+const classCardData = 'font-normal';
+const classCardDate = 'font-extralight';
```

Comment:
maybe group it as object?
```suggestion
const cardClassnames = {
  container: 'p-4 mb-5 bg-teal text-white rounded-lg',
  title: 'text-xl font-black',
  data: 'font-normal',
  date: 'font-extralight',
}
```

Reply:
already updated on https://github.com/RR-Ardhika/murojaah/pull/3/commits/201f7d6c9f8f8a8ef68bde319a412d68490fa2a8